mgc/discrim/discrimTwoSample.py

```diff
@@ -0,0 +1,170 @@
+from sklearn.metrics import euclidean_distances
+from ._utils import _CheckInputs
+import numpy as np
+import random
+from .base import DiscriminabilityTest
+from scipy._lib._util import MapWrapper
+
+
+class DiscrimTwoSample(DiscriminabilityTest):
+    r"""
+     A class that performs a two-sample test for whether the discriminability is different for that of
```

Comment:
Should be the following:
`A class that performs a two-sample test for discriminability.

Discriminability is used to test how different one data set is from another. More details can be described in [#1Dscr]_`

Reply:
https://github.com/neurodata/mgc/blob/8950dbc95e34818b226e177a9331fa37b7fbbaaf/mgc/discrim/discrim_two_samp.py#L9-L15